gravitee-am-common/src/main/java/io/gravitee/am/common/utils/ConstantKeys.java

```diff
@@ -75,11 +74,12 @@ public interface ConstantKeys {
     String AUTH_FLOW_CONTEXT_ATTRIBUTES_KEY = "authFlow";
 
     // MFA keys.
-    String MFA_SKIPPED_KEY = "mfaEnrollmentSkipped";
+    String MFA_STOP = "mfaStop";
+    String MFA_ENROLL_CONDITIONAL_SKIPPED_KEY = "mfaEnrollmentCanBeSkippedConditionally";
     String MFA_ENROLLMENT_COMPLETED_KEY = "mfaEnrollmentCompleted";
     String MFA_CHALLENGE_COMPLETED_KEY = "mfaChallengeCompleted";
     String STRONG_AUTH_COMPLETED_KEY = "strongAuthCompleted";
-    String ENROLLED_FACTOR_ID_KEY = "enrolledFactorId";
+    String SELECTED_ENROLL_FACTOR_ID_KEY = "enrolledFactorId";
```

Comment:
**issue**: We can't rename this constant key as it is used by a policy (a plugin) so if we want to keep this plugin compatible with AM 4.3.0, we have to keep ENROLLED_FACTOR_ID_KEY.
Other option is to keep the renaming and create a new version of the plugin